how-to/customize-workspace/public/second.manifest.fin.json

```diff
@@ -72,7 +72,7 @@
 					}
 				}
 			],
-			"authProviderId": "example"
+			"singleton": "example"

```

Comment:
should this be a boolean?

Reply:
Actually not necessary, the first enabled provider is used, and warned if more than one enabled, removed